portfolio/src/main/java/com/google/sps/servlets/UserLog.java

```diff
@@ -15,11 +18,17 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
-    response.setContentType("text/html");
+    JsonObject jsonObject = new JsonObject();
     if (userService.isUserLoggedIn()) {
-      response.getWriter().print("true");
+        String logoutURL = userService.createLogoutURL("/");
+        jsonObject.addProperty("checkifLoggedIn", true);
+        jsonObject.addProperty("linkforLoginLogout", logoutURL);
```

Comment:
since these strings are used multiple times, you might want to make them class constants. Also make sure they have consistent camel case: `checkIfLoggedIn` and `linkForLogInLogout`